packages/flutter/lib/src/widgets/form.dart

```diff
@@ -510,6 +510,7 @@ class FormField<T> extends StatefulWidget {
   ///
   /// This is called `value` in the [DropdownButtonFormField] constructor to be
   /// consistent with [DropdownButton].
+  /// The `initialValue` only affects the initial state when the form field is first built.
```

Comment:
Can you put an empty line of `///` above this?

Reply:
Oh also, it affects the state when FormField.reset is called!

Reply:
@justinmc Thank you for the feedback, addressed it